coalib/results/Diff.py

```diff
@@ -381,6 +381,19 @@ def add_lines(self, line_nr_before, lines):
         linediff.add_after = lines
         self._changes[line_nr_before] = linediff
 
+    def add_line(self, line_nr_before, line):
+        """
+        Adds line after the given line number.
+
+        :param line_nr_before: Line number of the line before the additions.
```

Comment:
`additions` ? there is going to be a single addition right?
also in the conflicterror it would say `Cannot add "lines"` but there is a single line.